core/common-util/src/main/java/uk/gov/gchq/gaffer/commonutil/stream/Streams.java

```diff
@@ -122,6 +125,35 @@ public static <T> Stream<T> toParallelStream(final T... array) {
         return StreamSupport.stream(Spliterators.spliterator(array, 0), true);
     }
 
+    /**
+     * Convert an iterable to a {@link java.util.stream.Stream}
+     * of {@link java.util.List}.
+     *
+     * @param iterable the input iterable
+     * @param size the size of the batch
+     * @param <T>  the type of object stored in the stream
+     * @return a stream containing the contents of the iterable in a list
+     */
+    public static <T> Stream<List<T>> toBatches(final Iterable<T> iterable, final int size) {
```

Comment:
It feels like this might be useful in other parts of Gaffer, but it doesn't seem to have any tests?

Reply:
This will partly be because the coverage wont be picked up if its tested in a different module